ui/lib/pki/addon/components/page/pki-role-details.ts

```diff
@@ -1,22 +1,24 @@
 import { action } from '@ember/object';
+import RouterService from '@ember/routing/router-service';
 import Component from '@glimmer/component';
-
-// interface Attribute {
-//   name: string;
-//   options?: {
-//     label?: string;
-//   };
-// }
+import FlashMessageService from 'vault/services/flash-messages';
+import { inject as service } from '@ember/service';
+import errorMessage from 'vault/utils/error-message';
 
 // TODO: pull this in from route model once it's TS
```

Comment:
I noticed this comment before - what does it mean? Just personal curiosity 😅

Reply:
In cloud-ui, since the route is also TS we can export it from that file and then import it, and it knows the shape based on what's returned in the `model` hook. Although here, we really just need the ember-data model which I haven't quite figured out how to do yet 😄

Reply:
oh wow! neat - thanks for explaining! I'll also keep a 👀  for how to do this 🤔